media/css/newsletter/newsletter.less

```diff
@@ -170,6 +179,11 @@ body {
 
             .subscription td {
                 min-width: 75px;
+                label {//this should not be displayed BUT if some one uses an addon or accessibility feature to display labels this will make sure they don't look weird on the page.
+                    font-size: 12px;
+                    color: #666;
+                    display: none;
```

Comment:
If an element is `display: none` then it usually means that screen readers won't acknowledge it. Might using the `.visually-hidden()` mixin be a good solution here?

Reply:
Done